hoomd/state.py

```diff
@@ -348,32 +363,74 @@ def set_snapshot(self, snapshot):
 
     @property
     def particle_types(self):
-        """list[str]: List of all particle types in the simulation state."""
+        """list[str]: List of all particle types in the simulation state.
```

Comment:
```suggestion
        """`list` [`str`]: List of all particle types in the simulation state.
```
Same for all the following